testsuite/features/step_definitions/common_steps.rb

```diff
@@ -680,7 +679,7 @@
 end
 
 # rubocop:disable Metrics/BlockLength
-When(/^I bootstrap (traditional|minion) client "([^"]*)" using bootstrap script with activation key "([^"]*)" from the (server|proxy)$/) do |client_type, host, key, target_type|
+When(/^I bootstrap minion client "([^"]*)" using bootstrap script with activation key "([^"]*)" from the (server|proxy)$/) do |host, key, target_type|
```

Comment:
Shouldn't we remove the word `client` here since we only reference it with the traditional client?

Reply:
+1 "minion client", while correct, sounds weird

Reply:
Ok, I can do this refactor too